Jenkinsfile.groovy

```diff
@@ -20,7 +20,7 @@ pipeline {
   agent { label "build && docker && linux" }
 
   environment {
-    NPM_TOKEN = credentials("npmjs_com_token")
+    NPM_TOKEN = credentials("NPM_TOKEN")
```

Comment:
I think @GermainBergeron had to fiddle something in the repo config for this env variable to work for us in plasma 🤔

Reply:
From what I see, he added a new secret in Jenkins, nothing more.